README.md

```diff
@@ -1,2 +1,22 @@
-# pass-bakery
-scala training project
+# Pass-Bakery
+Scala training project
+
+### Project Dependencies
```

Comment:
You title is one #, the next titles should be 2 ##, to keep the semantic meaning of the document